tests/test_netbox.py

```diff
@@ -82,7 +73,7 @@ class TestNetBoxInventory2(TestNBInventory):
     # only on NetBoxInventory2 and NetBox 2.8.9
     @pytest.mark.parametrize("version", ["2.8.9"])
     def test_vms_inventory(self, requests_mock, version):
```

Comment:
Would suggest to rename this test to `test_inventory_include_vms` as is better covers what it actually does.
The current name could be interpreted as if it is only testing VM's inventory, which is not the case.

Reply:
Done.  Makes sense.